test.html

```diff
@@ -2,9 +2,15 @@
 <html>
     <head>
         <meta charset="utf-8">
-        <title>Hello world!</title>
+        <title>Favourite Movie Quotes</title>
     </head>
     <body>
-        <h1>Hello World!</h1>
+        <h1>Favourite Movie Quotes</h1>
```

Comment:
I don't know these movies. Could you add some images of them?
https://www.w3schools.com/html/html_images.asp